src/creature.cpp

```diff
@@ -277,7 +277,7 @@ void Creature::startAutoWalk(const std::vector<Direction>& listDir)
 		return;
 	}
 
-	Player* player = getPlayer();
+	auto player = getPlayer();
```

Comment:
Could this be `const auto`
```suggestion
	const auto player = getPlayer();
```